src/views/pages/name/scottish-limited-partnership-name.njk

```diff
@@ -0,0 +1,150 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <form class="form" action={{ props.currentUrl }} method="post">
+
+      <input type="hidden" name="pageType" value={{ props.pageType }}>
+
+      {% include "includes/csrf_token.njk" %}
+
+      <h1 class="govuk-heading-xl">{{ i18n.scottishNamePage.title }}</h1>
+
+      <div class="govuk-form-group">
+        <h1 class="govuk-label-wrapper">
+          <label class="govuk-label govuk-label--l" for="partnership_name">
+            {{ i18n.scottishNamePage.whatIsName }}
+          </label>
+        </h1>
+
+        {{ govukInput({
+          classes: "govuk-input--width-40",
+          errorMessage: errors.scottish_partnership_name if errors,
+          id: "scottishPartnership_name",
+          name: "scottishPartnership_name",
+          value: scottishPartnership_name,
+          fieldset: {
+            legend: {
+              isPageHeading: false,
+              classes: "govuk-fieldset__legend--xl"
+            }
+          },
+          hint: {
+            text: i18n.scottishNamePage.whatIsNameHint
+          },
+          attributes : {
+            required : true,
+            "aria-describedby": "partnership_name-hint",
+            minLength: "3"
+          },
+          suffix: {
+              text: "Limited Partnership"
+            }
+          }) }}
+      </div>   
+
+      {{ govukInsetText({
+        html: '<p>' + i18n.namePage.existingPartnershipHint + '</p><p>' + i18n.namePage.sensitiveWordHint + '</p><p>' + i18n.namePage.existingTrademarkHint + '</p>'
+      }) }}
+
+     <div class="govuk-form-group">
+      <fieldset class="govuk-fieldset" aria-describedby="nameEnding-hint">
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+          <h1 class="govuk-fieldset__heading">
+          {{ i18n.scottishNamePage.nameEnding }}
+          </h1>
+        </legend>
+
+       {% set welsh %}
```

Comment:
Screenshot in the story suggests that Welsh options aren't required.

Reply:
Welsh option removed

Reply:
Still showing. Have you tested?

There is another Welsh option section in the HTML to remove.

Reply:
removed and tested